src/list.tsx

```diff
@@ -274,11 +276,21 @@ interface BasicRowContentProps extends CommonProps {
     atomicReading?: boolean;
 }
 
+type SwitchDisclosure = {
+    expanded: boolean; // is the related content expanded
+    controlsId?: string; // id of the related content (if any)
+    live?: 'off' | 'polite' | 'assertive'; // announcement channel when changing expanded
+    busy?: boolean; // blocks UI and sets aria-busy
+    showSpinner?: boolean; // shows spinner on the right
+    onLabelWhenExpanded?: string; // message when expanded = true
```

Comment:
why do we need a different label for the expanded state? can't this be controlled by Row users by changing the label when needed?